pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/utils/AuthTokenUtils.java

```diff
@@ -102,6 +102,9 @@ public static String createToken(Key signingKey, String subject, Optional<Date>
     public static byte[] readKeyFromUrl(String keyConfUrl) throws IOException {
         if (keyConfUrl.startsWith("data:") || keyConfUrl.startsWith("file:")) {
             try {
+                if (keyConfUrl.startsWith("file:")) {
+                    keyConfUrl = keyConfUrl.trim();
+                }
```

Comment:
Could you please help add a test to avoid regression?

Reply:
OK,I will fix

Reply:
Fixed,PTAL,thanks! @codelipenghui